source/includes/read-write-pref.scala

```diff
@@ -59,15 +64,29 @@ object ReadWritePref {
     val readPreference = ReadPreference.secondary(List(tag1, tag2, tag3).asJava)
 
     val database = mongoClient.getDatabase("test_database")
-        .withReadPreference(readPreference)
+                              .withReadPreference(readPreference)
     // end-tag-set
 
     // Instructs the library to distribute reads between members within 35 milliseconds
-    // of the closest member's ping time
-    // start-local-threshold
+    // of the closest member's ping time using client settings
+    // start-local-threshold-uri
     val connectionString = "mongodb://localhost:27017/?replicaSet=repl0&localThresholdMS=35"
     val client = MongoClient(connectionString)
-    // end-local-threshold
+    // end-local-threshold-uri
+
+    // Instructs the library to distribute reads between members within 35 milliseconds
+    // of the closest member's ping time using a URI option
+    // start-local-threshold-settings
+    val clusterSettings = ClusterSettings.builder()
+        .localThreshold(35, TimeUnit.MILLISECONDS)
+        .requiredReplicaSetName("repl0")
+        .build()
+
+    val client = MongoClient(MongoClientSettings.builder()
+        .applyConnectionString(ConnectionString("mongodb://localhost:27017/"))
+        .applyToClusterSettings(builder => builder.applySettings(clusterSettings))
```

Comment:
I saw a similar mistake on an earlier Scala PR: this will wipe out any ClusterSettings that are set on the connection string.  Instead you want something like:

```
.applyToClusterSettings(builder => builder.localThreshold(35, TimeUnit.MILLISECONDS))
```

and get rid of the `clusterSettings` variable entirely.